### PR TITLE
EDRD Sprint - 9 Development

### PR DESCRIPTION
**Pull request by deepakmulamalla**
EDRD-1031 | Remove EDRD Admin from EDRD PSP Queue
EDRD-1009 | Create additional fields to capture phone and email record for providers
EDRD-931 | As an EDRD Staff I want to be able to create reports to facilitate my work so that I do not require a superuser to create my reports
EDRD – 737 | [Enhancement] One-time dose flag on Medication object
EDRD – 1014 |  
EDRD-1017 | Update tooltip on deceased checkbox
EDRD – 1012 | As an EDRD Staff I want to see Case Type in all of the case list views so I can see if a case is New or Renewal
EDRD -1015 | When using the Create EDRD Case button, cases are being created without Medication Requests and before the workflow is completed
EDRD-1018 |  
EDRD-1010 | Update Account List View
EDRD - 1038 | Update case fits criteria label name
EDRD –  967 | EDRD Case Email Notification
EDRD-1007 | Error messages in the Provider Portal are not correct for Scenario 5 and 6
EDRD-1055(Bug) | When using the Create EDRD Case button, cases are being created without Medication Requests and before the workflow is completed
EDRD-992(Bug) | Risdiplam form fully filled out triggered error after signing off prescriber signature and submitting
EDRD-1052(Bug) | Eculizumab aHUS and PNH Funding Duration for Renewals is incorrect in the Medication record
EDRD-996(Bug) | Assessment Question Responses are not being populated from the submitted form for risdiplam & velaglucerase alfa
EDRD-1055(Bug) | When using the Create EDRD Case button, cases are being created without Medication Requests and before the workflow is completed
EDRD-992(Bug) | Risdiplam form fully filled out triggered error after signing off prescriber signature and submitting
EDRD-1012 | As an EDRD Staff I want to see Case Type in all of the case list views so I can see if a case is New or Renewal
EDRD-996(Bug) | Assessment Question Responses are not being populated from the submitted form for risdiplam & velaglucerase alfa
EDRD-985(Bug) | Special Authority has full access to Account  Relationships and is able to View, Edit, and Remove relationships established by EDRD
EDRD-988 | Deceased patient automation is updating  the MOH Decision Details in a Cancelled case as Denied

### CODE REVIEW

File: force-app/main/default/objects/Account/validationRules/EDRD_PhoneNumber_Validation.validationRule-meta.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ValidationRule xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>EDRD_PhoneNumber_Validation</fullName>
+    <active>true</active>
+    <errorConditionFormula>OR(
```
Comment (cgijeffolsen) [inline force-app/main/default/objects/Account/validationRules/EDRD_PhoneNumber_Validation.validationRule-meta.xml:5]: @deepakmulamalla Is this validation rule intended to apply to all accounts (SAT and EDRD)?

**Review by cgijeffolsen — CHANGES_REQUESTED** [review]
All permission changes are within scope for EDRD. All field changes appropriate to EDRD (none are required). List view changes only affect EDRD list views.  Flow and email changes are specific to EDRD.

Clarification needed: EDRD_PhoneNumber_Validation is not specific to EDRD users and will apply to all accounts. Please confirm this is expected.

File: force-app/main/default/objects/Account/validationRules/EDRD_PhoneNumber_Validation.validationRule-meta.xml (hunk 1 above)
Comment (deepakmulamalla) [reply force-app/main/default/objects/Account/validationRules/EDRD_PhoneNumber_Validation.validationRule-meta.xml]: @cgijeffolsen, I’ve updated the validation rule to include the EDRD user role check. Please review, and let me know if there are any further adjustments needed.

**Comment by deepakmulamalla** [comment]
> All permission changes are within scope for EDRD. All field changes appropriate to EDRD (none are required). List view changes only affect EDRD list views. Flow and email changes are specific to EDRD.
> 
> Clarification needed: EDRD_PhoneNumber_Validation is not specific to EDRD users and will apply to all accounts. Please confirm this is expected.

@cgijeffolsen, I’ve updated the validation rule to include the EDRD user role check. Please review, and let me know if there are any further adjustments needed.

**Review by cgijeffolsen — APPROVED** [review]
All changes now meet expectation and have no risks to SAT. Approved. @deepakmulamalla @RanadheerRG @NataliaNikishina

**Comment by deepakmulamalla** [comment]
> All changes now meet expectation and have no risks to SAT. Approved. @deepakmulamalla @RanadheerRG @NataliaNikishina

Thanks @cgijeffolsen, for approving the PR.

**Review by RanadheerRG — APPROVED** [review]
Approved @deepakmulamalla

**Comment by deepakmulamalla** [comment]
@NataliaNikishina, can you approve this PR?

**Comment by deepakmulamalla** [comment]
> Approved @deepakmulamalla

Thanks @RanadheerRG

**Review by NataliaNikishina — APPROVED** [review]
@deepakmulamalla approved

**Pull request merged**